src/NuGet.Core/NuGet.Commands/PackCollectorLogger.cs

```diff
@@ -25,6 +27,14 @@ public PackCollectorLogger(ILogger innerLogger, WarningProperties warningPropert
             _errors = new ConcurrentQueue<ILogMessage>();
         }
 
+        public PackCollectorLogger(ILogger innerLogger, WarningProperties warningProperties, WarningPropertiesCollection packageReferenceWarningPropertiesCollection)
```

Comment:
make the other constructor call this one with a default value for new parameters.

Reply:
Please check now.